e2e_tests/tests/cluster/test_slurm.py

```diff
@@ -191,6 +191,7 @@ def test_master_host() -> None:
 @pytest.mark.skipif(not torch.cuda.is_available(), reason="no gpu available")
 @pytest.mark.e2e_slurm
 @pytest.mark.e2e_pbs
+@pytest.mark.rbdebug  # XXX
```

Comment:
Drunken comment?

Reply:
I am `rb`, `debug` is what I'm doing, and XXX is my tag to not forget to revisit a line before landing it.